spec/generic_actor_spec.cr

```diff
@@ -7,6 +7,10 @@ class Counter
 
   @value = 0
 
+  def initialize(@name : String = "777")
+    print @name
```

Comment:
do we really need print here? I thought it was for debug purposes.

Reply:
I made print for debug, we can remove that, but initialize should stay.

Reply:
Removed print